src/tensorflow.cc

```diff
@@ -1763,21 +1763,21 @@ ModelState::AutoCompleteConfig()
           backend_config_->allow_soft_placement_,
           backend_config_->memory_limit_mb_, nullptr /* tftrt_config */,
           false /* auto_mixed precision */);
-    }
 
-    if (err != nullptr) {
-      std::string msg((err->msg_ == nullptr) ? "<unknown>" : err->msg_);
-      TRITONTF_ErrorDelete(err);
-      return TRITONSERVER_ErrorNew(
-          TRITONSERVER_ERROR_INTERNAL,
-          (std::string("unable to auto-complete model configuration for '") +
-           Name() + "', failed to load model: " + msg)
-              .c_str());
-    }
+      if (err != nullptr) {
+        std::string msg((err->msg_ == nullptr) ? "<unknown>" : err->msg_);
+        TRITONTF_ErrorDelete(err);
+        return TRITONSERVER_ErrorNew(
+            TRITONSERVER_ERROR_INTERNAL,
+            (std::string("unable to auto-complete model configuration for '") +
+             Name() + "', failed to load model: " + msg)
+                .c_str());
+      }
 
-    auto ach = AutoCompleteHelper(this, tritontf_model);
-    RETURN_IF_ERROR(ach.Fix());
-    RETURN_IF_ERROR(SetModelConfig());
+      auto ach = AutoCompleteHelper(this, tritontf_model);
+      RETURN_IF_ERROR(ach.Fix());
+      RETURN_IF_ERROR(SetModelConfig());
+    }
```

Comment:
Should we return an error that the file doesn't exist to propogate [here](https://github.com/triton-inference-server/tensorflow_backend/blob/b57301d25ce1e99672573d7afb92de987636f4de/src/tensorflow.cc#L1048)?

Reply:
The error will be returned by the usual model load pipeline. This will just help the model return early from auto-complete logic.